src/main/java/com/google/sps/data/Habit.java

```diff
@@ -1,15 +1,18 @@
-package main.java.com.google.sps.data;
+package com.google.sps.data;
```

Comment:
Small nit: You don't have to change it. But you technically don't have to namespace your code by com.google.sps.